.github/workflows/psgallery-push.yml

```diff
@@ -0,0 +1,30 @@
+name: Push PowerShell module
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  
+  publish:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 6.0.x
+
+    - name: Cake Action
+      uses: cake-build/cake-action@v1.4.0
+      with:
+        target: PowerShellPack
+
+    - name: Publish
```

Comment:
Can we do this step as a Cake task?
We are trying to keep things consistent, all CI/CD-related tasks are only in build.cake file. GitHub Actions are used only to run those tasks. So, no additional logic here to build or publish

Reply:
Yes I'm aware of this. I already tried to use [this addin](https://cakebuild.net/extensions/cake-powershell/) to execute the publish command but for some reason I run to [this issue](https://github.com/SharpeRAD/Cake.Powershell/issues/84). I also tried [this workaround](https://github.com/SharpeRAD/Cake.Powershell/issues/84#issuecomment-798572776) but same issue still occurs.

Reply:
` &loaddependencies=true` works for me

Reply:
I tried that on Linux. I'm not sure if it does make any difference, but I'll try on Windows.
Also I noticed using that made Cake importing a lot of packages (around 560Mb).

Reply:
MacOS build was failing after I added `Cake.Powershell` addin. I fixed it by separating PowerShell related tasks in a new cake file. This also fix the issue of cake importing a lot packages with `&loaddependencies=true` in PowerShell unrelated tasks.

Reply:
on Windows it works well in build.cake. We are going to use Windows agent in GitHub Action for this, so I think it would be good to have only one file

Reply:
Yes, it works well on Windows. The reason I had to separate it in a new cake file is because addins get loaded even they're not going to get used in a particular task. For example, Test task fails on MacOS because `Cake.Powershell` has some issues with that platform even though we don't need it. I'm not sure if there is any way to conditionally load addins.
[This is when I had all tasks in one cake file](https://github.com/Marusyk/grok.net/pull/49/commits/a3a04d5b53d445e0a00d4eb23fedcc1404ab49f6)